signac/core/synced_collection.py

```diff
@@ -0,0 +1,162 @@
+# Copyright (c) 2020 The Regents of the University of Michigan
```

Comment:
I'm not sure what you mean. Where are these copied from that would no longer be in the git history? The fact that some of the history will be hard to follow (because it's from files like the original synced_dict.py that will be removed once this is done) is a common problem with large-scale refactoring. As long as the original code was from within our code base I don't think we need to worry about this. If you're worried about crediting people appropriately w.r.t. authorship, we could start including a credits file in signac like we do in other projects.

Reply:
You're right, it's probably fine.